Description: variables(5) lines over 80 width

$ LC_ALL=C MANROFFSEQ='' MANWIDTH=80 man --warnings -E UTF-8 -l -Tutf8 -Z man/variables.5.def > /dev/null
 
 cause error.

Forwarded: no
Last-Update: 2014-01-11

--- net-snmp-5.7.2~dfsg.orig/man/variables.5.def
+++ net-snmp-5.7.2~dfsg/man/variables.5.def
@@ -84,7 +84,8 @@ A variant of this (typically used when w
 text, rather than running programs), is to combine the name and
 numeric subidentifier:
 .RS
- .iso(1).org(3).dod(6).internet(1).mgmt(2).mib-2(1).system(1).sysDescr(1)
+ .iso(1).org(3).dod(6).internet(1).mgmt(2).mib-2(1).system(1)
+ .sysDescr(1)
 .RE
 
 .SS Module-qualified OIDs
